### PR TITLE
BACKLOG-14186 add breadcrumb in content editor

### PR DESCRIPTION
**Pull request by Taybou**
## JIRA

https://jira.jahia.org/browse/BACKLOG-14186

## Tests

The following are included in this PR
- [ ] Unit Tests (Most changes _should_ have unit tests)
- [ ] Integration Tests

## Checklist

I have considered the following implications of my change: 

- [ ] Security (in particular for changes to authentication, authorization, data fetching, ...)
- [ ] Performance
- [ ] Migration
- [ ] Code maintainability

## Documentation

- [ ] Inline documentation
- [ ] Internal Documentation (wiki)
- [ ] User-facing Documentation

### CODE REVIEW

File: src/javascript/EditPanel/header/ContentBreadcrumb/ContentPath/ContentPath.container.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    const {data, error} = useQuery(GetContentPath, {
+        variables: {
+            path: path,
+            language
+        }
+    });
+
+    const handleNavigation = path => {
+        dispatch(cmGoto({
+            mode: mode || 'content-folders',
```
Comment (dgriffon) [inline src/javascript/EditPanel/header/ContentBreadcrumb/ContentPath/ContentPath.container.jsx:27]: What happen if `cmGoto` is not available (like content editor loaded without jContent?)
We need to handle that case

File: src/javascript/EditPanel/header/ContentBreadcrumb/ContentPath/ContentPath.container.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+import {cmGoto} from '~/JContent.redux-actions';
+import {GetContentPath} from './ContentPath.gql-queries';
+import ContentPath from './ContentPath';
+
+const ContentPathContainer = ({path}) => {
+    const dispatch = useDispatch();
+
+    const {mode, language} = useSelector(state => ({
+        mode: state.jcontent.mode,
```
[the remark this answers is not in the document]
Comment (Taybou) [reply src/javascript/EditPanel/header/ContentBreadcrumb/ContentPath/ContentPath.container.jsx:14]: yes

[the remark this answers is not in the document]
Comment (Taybou) [reply src/javascript/EditPanel/header/ContentBreadcrumb/ContentPath/ContentPath.container.jsx:14]: and no

[the remark this answers is not in the document]
Comment (Taybou) [reply src/javascript/EditPanel/header/ContentBreadcrumb/ContentPath/ContentPath.container.jsx:14]: will be handle it in the next subtask

File: src/javascript/EditPanel/header/ContentBreadcrumb/ContentPath/ContentPath.container.jsx (hunk 1 above)
Comment (Taybou) [reply src/javascript/EditPanel/header/ContentBreadcrumb/ContentPath/ContentPath.container.jsx:27]: if there is no jContent, we will not be possible to redirect content anywhere

Comment (dgriffon) [reply src/javascript/EditPanel/header/ContentBreadcrumb/ContentPath/ContentPath.container.jsx:27]: if `cmGoto` not present, this call will fail, you need to be sure that cmGoto is available before building the link.

Comment (Taybou) [reply src/javascript/EditPanel/header/ContentBreadcrumb/ContentPath/ContentPath.container.jsx:27]: I'll create task to handle this case

**Pull request merged**